### PR TITLE
refactor(action-chain): use private keyword in constructor argument list

### PR DESCRIPTION
**Pull request by Guria**

### CODE REVIEW

File: packages/node_modules/action-chain/src/ActionBase.ts [hunk 1]
```diff
@@ -6,22 +6,20 @@ export class StopExecution {
...
   }
 }
+
+interface I<Context, InitialValue, Value = InitialValue>
+  extends ActionBase<Context, InitialValue, Value> {
+  (value: string): string
+  displayName: string
+}
+
 export class ActionBase<Context, InitialValue, Value = InitialValue> {
```
Comment (Guria) [inline packages/node_modules/action-chain/src/ActionBase.ts]: Value generic isn't used elsewhere. Do we need it?

File: packages/node_modules/action-chain/src/ActionBase.ts [hunk 2]
```diff
@@ -65,21 +60,22 @@ export class ActionBase<Context, InitialValue, Value = InitialValue> {
...
           executionId: executionContext.__execution.executionId,
-          actionName: func.displayName,
+          actionName: instance.displayName,
         })
       }
 
       return returnValue
-    }, this) as any
+    },
+    this) as any
```
Comment (Guria) [inline packages/node_modules/action-chain/src/ActionBase.ts]: hmm, weird formatting here

File: packages/node_modules/action-chain/src/ActionBase.ts (hunk 1 above)
Comment (Guria) [reply packages/node_modules/action-chain/src/ActionBase.ts]: tried in another branch
https://github.com/cerebral/overmind/compare/ac-ref2
should I replace PR with this variant?

Comment (christianalfoni) [reply packages/node_modules/action-chain/src/ActionBase.ts]: Yeah, this should be cleaned up :) I am working on a PR now which replaces `Context` and `Providers` with the term `Effects`. This gives us two things:

**Comment by Guria** [comment]
@christianalfoni rebased again

**Pull request merged**